### DISCUSSION TITLE
Application was unable to launch correctly 0xc0000142

### DISCUSSION THREAD

**Discussion by hotmedal** [carla-simulator/carla#4662] [Q&A]
Downloaded the 0.9.12 zip for Windows, unzipped it to E:\WindowsNoEditor, tried launching the CarlaUE4.exe directly, then by command prompt using -quality-level=Low, but always got the same error message (no output in terminal):

CarlaUE4-Win64-Shipping.exe - Application Error
The application was unable to start correctly (0xc0000142). Click OK to close the application. 

I watched youtube videos and tried the commands in Administrator cmd:
for %i in (%windir%\system32*.dll) do regvr32.exe / s%i
for %i in (%windir%\system32*.ocx) do regsvr32.exe / s%i

My windows locale is English US. I have Python 3.9.7 installed. I ran pip on a couple of requirements.txt found in the PythonAPI folders (numpy is defninitely installed). 

Any ideas what I'm doing wrong?

Edit: System is Core i5-3570, Radeon RX 570, 8 GB RAM, DirectX 12, Visual C++ redistributables from 2008 all the way to 2015-2019 (could the old ones be conflicting? They are all installed)

**Comment by hitabm** [comment]
I think you have to check your system hardware compatibility at the first. Then check DirectX , Visual C++ and GPU drivers to be installed correctly.

**Reply by hotmedal** [reply]
System is Core i5-3570, Radeon RX 570, 8 GB RAM, DirectX 12, Visual C++ redistributables from 2008 all the way to 2015-2019 (could the old ones be conflicting? They are all installed), Radeon drivers updated. Games like Star Wars Jedi of the Fallen Order and Fortnite run on it.

If hardware is lacking, shouldn't it work with low graphics settings?

**Reply by hitabm** [reply]
Visual C++ versions regularly do not conflict each other, so I don't think that would be your problem. But as I see in you specs, your RAM capacity is low and your CPU may not satisfy CARLA requirements. When you try to run low graphics mode, it reduces GPU processes and adds some to CPU duties! So, I'm not sure enough about saying this, but you have to try normal running of CARLA it may work. If it didn't, I'm sorry to say you will not be able to run CARLA on your PC. (As a plan B, search and try to run CARLA on Google Colab similar to [this ](https://github.com/MichaelBosello/carla-colab)repo)

**Reply by hotmedal** [reply]
I downloaded version 0.9.9 and it works perfectly. So just a problem with 0.9.12 maybe as pointed out by joluko below.

**Comment by joluko** [comment]
I have the  same problem with Carla_0.9.12 but the previous version is running without any issues. Any idea what causes the problem?

#4609 

Windows 10 Enterprise
Intel(R) Xeon(R) CPU W3670 @ 3.20GHz 3.20 GHz 12GB Ram (driver version: 10.0.19041.546)
Nvidia Quadro K2000 (driver version: 30.0.14.7212)

**Reply by hotmedal** [reply]
Yes, it appears only to be a problem with 0.9.12, since 0.9.9 is working fine for me.

**Reply by hitabm** [reply]
@hotmedal Please check [these ](https://github.com/carla-simulator/carla/issues/4609#issuecomment-924064063)too.
